From 38b5ef03fd17dc439b90a43d6c7209ee9ce41308 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Kamil=20P=C3=A1ral?= <kamil.paral@gmail.com>
Date: Wed, 10 Oct 2018 19:17:00 +0200
Subject: [PATCH 0484/2346] readme: only adjust hard limit in systemd (#20)

Instead of increasing both soft and hard fileno limit in systemd
instructions, increase just the hard limit. Soft limit needs to stay at
1024 for compatibility with programs using select() instead of newer
poll()/epoll(), otherwise such programs might fail.
---
 README.esync | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/README.esync b/README.esync
index 11d86563a10..6520f2da564 100644
--- a/README.esync
+++ b/README.esync
@@ -23,7 +23,7 @@ On distributions using systemd, the settings in `/etc/security/limits.conf`
 will be overridden by systemd's own settings. If you run `ulimit -Hn` and it
 returns a lower number than the one you've previously set, then you can set
 
-DefaultLimitNOFILE=1048576
+DefaultLimitNOFILE=1024:1048576
 
 in both `/etc/systemd/system.conf` and `/etc/systemd/user.conf`. You can then
 execute `sudo systemctl daemon-reexec` and restart your session. Check again
-- 
2.47.0

